app/build.gradle

```diff
@@ -20,7 +20,7 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled true
+            minifyEnabled false
```

Comment:
You should have minifyEnabled to true for buildTypes release in order to enable code shrinking, obfuscation, and optimization .